lantern_test.go

```diff
@@ -56,7 +57,7 @@ func testProxiedRequest(proxyAddr string, socks bool) error {
 		URL: &url.URL{
 			Scheme: "http",
 			Host:   "www.google.com",
-			Path:   "http://www.google.com/humans.txt",
+			Path:   "/humans.txt",
```

Comment:
Drive-by fix while I was trying to see if I could get the tests to work.  Unless there's something unique going on here, URL.Path refers to the sub-path under a host.